src/main/sensors/esc_sensor.c

```diff
@@ -135,45 +136,45 @@ bool escTelemetryInit(void)
     portOptions_t options = (SERIAL_NOT_INVERTED);
 
     // Initialize serial port
-    escTelemetryPort = openSerialPort(portConfig->identifier, FUNCTION_TELEMETRY_ESC, escTelemetryDataReceive, ESC_TLM_BAUDRATE, MODE_RX, options);
+    escSensorPort = openSerialPort(portConfig->identifier, FUNCTION_TELEMETRY_ESC, escSensorDataReceive, ESC_SENSOR_BAUDRATE, MODE_RX, options);
```

Comment:
That's been changed in the configurator. ESC feedback and GPS are now in a 'Sensors' drop down: 

https://github.com/betaflight/betaflight-configurator/blob/master/tabs/ports.js#L34

The name is still 'telemetry', but we hadn't agreed on a name when I did the reorg of the configurator tab. Also, I am not sure that 'ESC sensor' is a good user facing name, I'd probably rather call it 'ESC feedback'.